packages/direction-mixin/direction-mixin.ts

```diff
@@ -0,0 +1,39 @@
+import { DirectionClass } from './direction-class';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+const appliedMixins = new WeakMap();
```

Comment:
Do we really need a `WeakMap` over here?
If I understood correctly, we use only one `DirectionMixin`  and a list of extended classes? 🤔

Reply:
Changed to `Set`, thanks.